src/components/CsvBlockTable.tsx

```diff
@@ -0,0 +1,357 @@
+            </A>
+            . This ID can be
+            used to look up the location information in the Location file.
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>represented_country_geoname_id</Td>
+          <Td>integer</Td>
+          <Td>
+            The represented country is the country which is represented by
+            users of the IP address. For instance, the country represented by
+            an overseas military base. This column contains a unique
+            identifier for the network&apos;s registered country as specified
+            by
+            {' '}
+            <A
+              href="https://www.geonames.org/"
+              rel="nofollow noopener noreferrer"
+              target="_blank"
+            >
+              GeoNames
+            </A>
+            . This ID can be
+            used to look up the location information in the Location file.
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>is_anonymous_proxy</Td>
+          <Td>boolean</Td>
+          <Td>
+            <Strong>Deprecated.</Strong>
+            {' '}
+            Please see our
+            {' '}
+            <A
+              href="https://www.maxmind.com/en/geoip2-anonymous-ip-database"
+              rel="nofollow noopener noreferrer"
+              target="_blank"
+            >
+              GeoIP2 Anonymous IP database
+            </A>
+            {' '}
+            to determine whether the IP address is used by an anonymizing
+            service.
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>is_satellite_provider</Td>
+          <Td>boolean</Td>
+          <Td>
+            <Strong>Deprecated.</Strong>
+          </Td>
+          { !isEnterprise && (
+            <Td>
+              <ServiceTag
+                text="Country"
+              />
+              <ServiceTag
+                text="City"
+              />
+            </Td>
+          )}
+        </Tr>
+        <Tr>
+          <Td>postal_code</Td>
+          <Td>string</Td>
+          <Td>
+            A postal code close to the user&apos;s location. We return the
+            first 3 characters for Canadian postal codes. We return the first
+            2-4 characters (outward code) for postal codes in the United
+            Kingdom.
```

Comment:
On the current site we have the following info for the number of char returned for each country. Does that need to be added here?
```A postal code close to the user's location. For the following countries, we return partial postal codes with the number of characters indicated below:

United States: 5
Canada: 3
United Kingdom: 2-4
Brazil: 5
Ireland: 3
Japan: 7 (specified for the first 6. The last digit defaults to 1)
Netherlands: 4
Portugal: 7 (accurate for the first 4. The last 3 often defaults to -001)
Singapore: 2

Reply:
I think I see where the two different descriptions came from. On the https://dev.maxmind.com/geoip/geoip2/geoip2-city-country-csv-databases/ page it shows the text I added above for `postal_code`, but on the https://dev.maxmind.com/geoip/geoip2/geoip2-enterprise-csv-database/ page it shows the text you had originally. I am not sure if they need to remain different?

Reply:
Also just FYI the font needs to match up
![image](https://user-images.githubusercontent.com/47989320/114716574-2ddebe80-9d02-11eb-9d0f-10670c1f2b04.png)